sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOrcScan.scala

```diff
@@ -2123,7 +2123,7 @@ class MultiFileCloudOrcPartitionReader(
         }
       } catch {
         case e: Throwable =>
-          hostBuffers.foreach(_.hmb.safeClose())
+          hostBuffers.foreach(_.hmbs.foreach(_.safeClose()))
```

Comment:
Not technically part of this PR, but couldn't `safeClose` throw and then we don't see `e`? We should probably also use the Seq variant of `safeClose(e)`